packages/nimble-components/src/card/template.ts

```diff
@@ -1,4 +1,9 @@
 import { html } from '@microsoft/fast-element';
 import type { Card } from '.';
 
-export const template = html<Card>`<slot></slot>`;
+export const template = html<Card>`
+    <section aria-labelledby="titleSlot">
```

Comment:
Here's how this appears in the accessibility tree for the storybook example:
![image](https://github.com/ni/nimble/assets/35350751/b4a8b872-7912-4ccb-bf15-3cc96d2d3fa4)

Reply:
Is there anything here that I can/should add a unit test for? I looked for unit tests in the other components for confirming the `role` or the behavior of `aria-labledby` but I didn't see anything that looked similar.

Reply:
I can't think of anything worthwhile to test here.

For some components, we test that the correct `role` is applied, but that's coming for free for the card because of using the `section` element. We also tend to write tests for dynamic ARIA attributes or ARIA attributes that need to get proxied from the host element to a different element within the template, but neither of those are the case for the card.